### PR TITLE
Stabilize `[env]` sections

### PR DESCRIPTION
**Pull request by joshtriplett**
`[env]` sections seem to work as advertised.

### CODE REVIEW

**Comment by alexcrichton** [comment]
@rfcbot fcp merge 

👍

File: src/doc/src/reference/unstable.md [hunk 1]
```diff
@@ -1125,32 +1125,6 @@ The `-Z future-incompat-report` flag enables the creation of a future-incompat r
 for all dependencies. This makes users aware if any of their crate's dependencies
 might stop compiling with a future version of Rust.
 
-### configurable-env
```
Comment (ehuss) [inline src/doc/src/reference/unstable.md:1128]: Can you add a redirect at the bottom of the page (in the script tag) for the `#configurable-env` fragment? The target should be "config.html#env"

**Comment by ehuss** [comment]
Ping @joshtriplett, not sure if you're still interested in pushing this.  If you want to fix the small nit about the redirect, I don't think there are any other blockers.

**Comment by joshtriplett** [comment]
(Currently fixing the merge conflicts)

**Comment by joshtriplett** [comment]
I believe this is now ready to merge, and has passed FCP.

**Pull request merged**